### ISSUE TITLE
Allow 2 tile wide art pieces

### ISSUE THREAD

**Issue by Mooses2k** [Mooses2k/SecretHistories#140]
Rooms size 4 tiles wide and larger should always have an even number of tiles in order for 2-wide tiles to be used and line-up nicely for better visuals. [basically DONE]

Only one dimension needs to be even, the other can still be allowed to be odd, but then has to place the single tile in the center, for instance, 4x5 room. 4 side is two 2-tiles. 5 side is 2-tile, 1-tile (perhaps with doorway), 2-tile. [basicalily DONE]

 it's about mesh assets mainly.
Our tile size is 1.5m.
However, most everything in the game is built in 2x2 of those segments.
So really rooms appear to be multiples of 3x3m tiles, 1.5m x 2.
However, art can still only handle 1.5m segments.
Our artists have made very nice 3x3m wall, floor, and ceiling segments.
And those should be used instead of the 1.5m segments where it makes sense
So two parts:
a) allow that to happen
b) a couple suggestions on where some of that art should go
In other words, rooms with pillars should used the domed ceiling asset and the arch on the wall assets

Relates to and is precursor for #17 and #123
